trainings/sourcegraph-101/test-exercises.md

```diff
@@ -0,0 +1,38 @@
+# Sourcegraph 101 test exercises
```

Comment:
@emchap that makes sense. I'll think on it, maybe for now letting it sit. If this is how we'll be doing 102 and Admin trainings though then yes, we should update `template.md` because we'd want templates for building courses to follow the same pattern when possible.

I'll think this one over, and this comment is non-blocking for the PR.